sympy/stats/tests/test_discrete_rv.py

```diff
@@ -50,6 +51,17 @@ def test_Poisson():
     assert isinstance(E(2*x, evaluate=False), Expectation)
     # issue 8248
     assert x.pspace.compute_expectation(1) == 1
+    try:
+        import numpy as np
+    except ImportError:
+        skip("numpy not installed")
+    y = Poisson('y', np.float64(4.72544290380919e-11))
```

Comment:
I'm sorry. I should have pointed it out yesterday, but please make sure to leave the issue number as a comment.

Reply:
I will do that. Could you briefly explain when do newly added tests need an issue number accompanying them?

Reply:
As with any code, making it easy to read is essential. This means reducing cognitive load. For instance, testing that the expected value of $Poisson(\lambda)$ is $\lambda$ seems natural (so I don’t think leaving a comment is necessary). On the other hand, someone reading a test case involving an unfamiliar value like `4.72544290380919e-11` might wonder, "Why this number?" While there are no strict rules, I believe comments are necessary to ensure that someone unfamiliar with the context won't get stuck while reading the code.

Reply:
That makes complete sense! Thank you for the explaination, I will keep this in mind with my further contributions.

Reply:
Should this be `4.725442903809197e-11` ? (ending in 7)

`4.72544290380919e-11` was the case that worked even before this patch.

Reply:
I assume the exact numbering is not the issue here, more about the handling of E(x) queries.

Reply:
I overlooked it. I'll fix it.

Reply:
Even I missed that, sorry.